charts/prometheus-kafka-exporter/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v2
-appVersion: "v1.2.0"
+appVersion: "v1.3.1"
```

Comment:
Fixing it here is not really accurate as the "latest" version is used in the values file. It should be better if the same version is declared in the values file.

Reply:
Yeah, I know. I just wanted it to be up to date. 

Wouldn't it be better to leave it as `tag: latest` so that we don't need to manually update the `tag` every time some new release comes out?

Reply:
From my point of view, it's a best practice to always fix the tag to be used by the helm chart. We ensure that the current chart is working with the given tag and we don't introduce breaking changes without knowing it.
Most of the other exporter's helm chart in this repository have a fixed tag version.

Reply:
Makes sense! I'll fix it then 😄